demos/notebooks/demo_VST.ipynb

```diff
@@ -118,8 +118,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "if play_movies:\n",
-    "    movie_gat.play(magnification=4, q_max=99.8)"
+    "#if play_movies:\n",
```

Comment:
I think the play_movies boolean makes commenting out lines like this now unnecessary